src/physics/utils/phys_comp.F90

```diff
@@ -158,12 +151,31 @@ subroutine phys_init()
       ! Confirm that the suite parts are as expected
       do i_group = 1, size(suite_parts)
          if (.not. any(suite_parts(i_group) == suite_parts_expect)) then
-            write(errmsg, *) 'phys_init: SDF suite groups MUST be ',             &
+            write(errmsg, *) 'phys_register: SDF suite groups MUST be ',      &
```

Comment:
Should this check come before the call to `cam_ccpp_physics_register`?  It would seem silly to do a bunch of constituent registering work just to then die because one of the SDF groups was bad.

Reply:
ooh yes - flipped the ordering!